keyboardClass.js

```diff
@@ -0,0 +1,241 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable indent */
+class Keyboard {
+    constructor(lang, langAdd, buttonList, container) {
+        this.language = localStorage.getItem('lang') !== null ? localStorage.getItem('lang') : lang;
+        this.languageAlter = this.language === 'en' ? 'ru' : 'en';
+        this.btnList = buttonList;
+        this.domElement = container;
+        this.upCase = false;
+        this.shiftPressed = false;
+        this.capsPressed = false;
+        this.ctrlPressed = false;
+        this.altPressed = false;
+        this.keyboardRows = [];
+        for (let row = 0; row <= 4; row += 1) {
+            this.keyboardRows[row] = document.createElement('DIV');
+            this.keyboardRows[row].classList.add('keyboard__row');
+            this.domElement.appendChild(this.keyboardRows[row]);
+        }
+        this.targetOfKeyboard = document.getElementById('textarea');
+        this.domElement.addEventListener('click', (event) => {
+            let keyCode = null;
+            if (event.target.tagName === 'BUTTON') keyCode = event.target.id;
+            else if (event.target.tagName === 'SPAN') keyCode = event.target.parentElement.id;
+            else return;
+
+            if (keyCode === 'ShiftLeft' || keyCode === 'ShiftRight' || keyCode === 'AltLeft' || keyCode === 'AltRight' || keyCode === 'ControlLeft' || keyCode === 'ControlRight') return;
+            this.triggerKeyboardEvent(document, 'keydown', keyCode);
+            setTimeout(() => { this.triggerKeyboardEvent(document, 'keyup', keyCode); }, 200);
```

Comment:
ahah that's nice but not correct. If you wanted to make the animation delayed play with css. That's not working for fast key press for example